terraform/aspen_batch_iam.tf

```diff
@@ -1,6 +1,65 @@
+data "aws_secretsmanager_secret" "gisaid-download-credentials" {
+  name = "gisaid-download-credentials"
+}
+
 resource "aws_iam_role" "aspen-batch-jobs-role" {
   name = "aspen-batch-jobs-role"
   assume_role_policy = templatefile("${path.module}/iam_templates/trust_policy.json", {
     trust_services = ["ecs-tasks"]
   })
 }
+
+
+resource "aws_iam_policy" "aspen-batch-jobs-policies" {
+  name = "aspen-batch-jobs-policies"
+  policy = <<-POLICY
+  {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": "secretsmanager:GetSecretValue",
+              "Resource": [
+                  "${data.aws_secretsmanager_secret.aspen_config.arn}",
+                  "${data.aws_secretsmanager_secret.gisaid-download-credentials.arn}"
+              ]
+          },
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "batch:SubmitJob"
+              ],
+              "Resource": [
+                  "arn:aws:batch:*:*:job-definition/${aws_batch_job_definition.aspen-batch-job-definition.name}",
+                  "arn:aws:batch:*:*:job-queue/${aws_batch_job_queue.aspen-batch-job-queue.name}"
+              ]
+          },
+          {
+              "Effect": "Allow",
+              "Action": "rds:DescribeDBInstances",
+              "Resource": [
+                  "${aws_db_instance.db.arn}"
+              ]
+          },
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "s3:GetObject",
+                  "s3:PutObject"
+              ],
+              "Resource": [
+                  "arn:aws:s3:::${aws_s3_bucket.aspen-data.bucket}/raw_gisaid_dump/*",
+                  "arn:aws:s3:::${aws_s3_bucket.aspen-data.bucket}/processed_gisaid_dump/*",
+                  "arn:aws:s3:::${aws_s3_bucket.aspen-data.bucket}/aligned_gisaid_dump/*"
```

Comment:
would be nice to move these key names to their own local variables eventually, just in case we need to use them in any other resource down the road.

Reply:
Yeah, I don't know if there's a way to share strings between terraform and python... @jgadling ?